ibm_catalog.json

```diff
@@ -58,6 +58,40 @@
                     }
                 ]
             },
+            "configuration" : [
+              {
+                "key": "ibmcloud_api_key"
+              },
```

Comment:
I see no value to adding this here? Perhaps add it when we plan to created a sorted list of vars?